AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStoreAdmin/src/MutationStateStructures.dfy

```diff
@@ -422,46 +501,61 @@ module {:options "/functionSyntax:4" } MutationStateStructures {
     :- NeedOutcome(
          Structure.BRANCH_KEY_RESTRICTED_FIELD_NAMES !! EncryptionContextKeys,
          () => Types.KeyStoreAdminException(
-             message := "Invalid Mutation Token: MUST NOT model Item specific fields!"
+             message := "Invalid Mutation Commitment: MUST NOT model Item specific fields!"
            )
        );
-
     Outcome.Pass
   }
 
   function MutationsInputJson?(
```

Comment:
Ditto here. This is tricky to follow without knowing what needs/conditions are being met (i.e. spec) but I trust this is what's needed